### ISSUE TITLE
HOW TO STILL USE SNAPCHAT WITH NEW SAFETYNET

### ISSUE THREAD

**Issue by EduardJipa** [Juxstin/EdXposed-SnapChat-Bypass#32]
Basically just completely uninstall edxposed & magisk, which will delete TWRP too, then boot & reset the caches and delete app data for:

- Google Play Store
- Google Play Services
- Google Services Framework
- Google Pay
- Snapchat

Reboot, log in to snapchat (It may fail a few times, worked 3rd try in for me)

May work for other apps such as google pay or netflix

(Also sorry if this is the wrong section, just wanted to help others in the same situation as I was)

**Comment by crooowned** [comment]
Just disable EdXposed Framework in Magisk, and you should pass SafetyNet...
Then reenable and reboot.
I dont think you need to Uninstall complete Magisk since Magisk alone passes SafetyNetright?

**Comment by EduardJipa** [comment]
Nope, no longer works; looks like magisk / some remnant of edxposed trips safetynet.

**Comment by poweromania** [comment]
> Nope, no longer works; looks like magisk / some remnant of edxposed trips safetynet.

is not because magisk or riru.. i have both installed and safetynet is ok

**Comment by letswritecodes** [comment]
I'm using snapchat even though my safetynet fails.. with edxposed installed

**Comment by poweromania** [comment]
Which version of Snapchat?

Pe vineri, 1 mai 2020, Varun K <notifications@github.com> a scris:

**Comment by EduardJipa** [comment]
> > Nope, no longer works; looks like magisk / some remnant of edxposed trips safetynet.
> 
> is not because magisk or riru.. i have both installed and safetynet is ok

Looks like it's device specific, as I had each of these components and no more. Complete fresh install. I'm using a Xiaomi 9T on Snapchat build 10.85.5.0

**Comment by crooowned** [comment]
i can confirm, installing this version of edXposed trips safetynet! even with coreonly enabled!
Maybe we need a new edxposed version with snap bypass 🌵

**Comment by Ti64CLi** [comment]
We need the owner of this edXposed to update it to last edxposed canary, which seems to pass safetynet just fine

**Comment by AlphaDream31** [comment]
My snapchat also works fine without safety net (1 month and going)
